sentry-ruby/lib/sentry/session.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Sentry
+  class Session
+    attr_reader :started, :status
+
+    # TODO-neel add :crashed after adding handled
+    STATUSES = %i(ok errored exited)
+    AGGREGATE_STATUSES = %i(errored exited)
+
+    def initialize
+      @started = Sentry.utc_now
+      @status = :ok
+    end
+
+    def update_from_event(event)
+      return unless event.is_a?(Event) && event.exception
```

Comment:
yea you're right, I did it this way for consistency with python but it just adds confusion, will change

Reply:
but I will keep `update_from_exception` since one #1743 is implemented, we will decide if the status is `:crashed` or `:errored` from the `mechanism` property.